### ISSUE TITLE
getDynamicStyles is no longer exported

### ISSUE THREAD

**Issue by vogre** [cssinjs/jss#1307]
__Expected behavior:__
Building a project using `import { createStyles, makeStyles } from '@material-ui/core';` succeeds.

__Describe the bug:__
When running `npm run build`.
```
Creating an optimized production build...
Failed to compile.

./node_modules/@material-ui/styles/esm/makeStyles/makeStyles.js
Attempted import error: 'getDynamicStyles' is not exported from 'jss'.

```

__Versions (please complete the following information):__
 - jss: 10.1.0
 
Usage in material-ui: https://github.com/mui-org/material-ui/blob/f27c78faa33f863199b8984990cb3673aecf071d/packages/material-ui-styles/src/makeStyles/makeStyles.js#L2

**Comment by kof** [comment]
I was hoping noone is using it, going to release a 10.1.1 with a patch

**Comment by viperz0r** [comment]
Same problem here, material-ui is using it

**Comment by kof** [comment]
Should be fixed now  in 10.1.1
